winml/lib/Api.Core/FeatureDescriptorFactory.cpp

```diff
@@ -438,31 +440,29 @@ CreateImageFeatureDescriptor(
   auto alpha_mode = format_info.second;
 
   // color space gamma value
-  auto color_space_gamma_value = FetchMetadataValueOrNull(metadata, c_color_space_key);
-  auto color_space_gamma = CreateImageColorSpaceGamma(color_space_gamma_value);
+//  auto color_space_gamma_value = FetchMetadataValueOrNull(metadata, c_color_space_key);
```

Comment:
Why hardcode the creation of ImageFeatureDescriptor to 0-255 Nominal Range and SRGB color space gamma?